environment/src/main/java/com/toggl/environment/di/EnvironmentModule.kt

```diff
@@ -84,6 +94,31 @@ object EnvironmentModule {
             installLocation
         )
     }
+
+    @Provides
+    @ApiAuthCredentials
+    fun authCredentials(): String {
+        //TODO: actually get the current logged in user api token
+        val validApiToken = "api token"
+        val authStringBytes = validApiToken.toByteArray(charset = Charsets.UTF_8)
+        return Base64.encodeToString(authStringBytes, Base64.DEFAULT)
+    }
+
+    @Provides
+    fun authInterceptor(@ApiAuthCredentials authString: String?): AuthInterceptor = AuthInterceptor(authString)
+
+    @Provides
+    @BaseApiUrl
+    fun baseEndpointUrl(): String =
+        if (BuildConfig.BUILD_TYPE.contentEquals("release")) "https://mobile.toggl.com"
```

Comment:
Let's create our own wrapper that does not give me gag reflex when we read it 😂 

```suggestion
        if (BuildConfig.buildTypeIsRelease) "https://mobile.toggl.com"
```

Or something like that